File: note/Python_algo/P002.py
# 평균 구하기
n = input()
mylist = list(map(int, input().split()))    # map() 함수는 리스트의 요소를 지정된 함수로 처리해주는 함수
mymax = max(mylist)
sum = sum(mylist)
print(sum * 100 / mymax / int(n))

# 입력
# 5
# 1 2 4 8 16

# 출력
# 38.75

File: note/cpp_algo/기타/P002.cpp
#include <iostream>

using namespace std;

int main() 
{
    int N = 0;
    int A[1000];
    cin >> N;

    for (int i = 0; i < N; i++) {
        cin >> A[i];
    }

    long sum = 0;
    long max = 0;   // int로 선언하면 오답이 나올 수 있음

    for (int i = 0; i < N; i++) {
        if (A[i] > max) {
            max = A[i];
        }
        sum = sum + A[i];
    }

    double result = sum * 100.0 / max / N;
    cout << result << "\n";


}